android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -200,7 +203,7 @@ class ModelViewer : android.view.View.OnTouchListener {
     /**
      * Renders the model and updates the Filament camera.
      */
-    fun render() {
+    fun render(frameTimeNanos: Long) {
```

Comment:
Can we do something reasonable when no argument is passed?

Reply:
Yes, so it actually does something reasonable when you pass 0 -- it disables frame-pacing. I didn't want to have a version that takes nothing because I think people will just be lazy and use that instead of passing the value from choreographer.

This can make a big difference on the experience.

Reply:
Okay but technically API changes like this merit a new major version number.